README.md

```diff
@@ -18,3 +18,61 @@ _yarn_:
 ```bash
 yarn add @vesselapi/react-vessel-link
 ```
+
+### Usage
+The `vessel-link` modal can be triggered using two methods:
+1. Through the `VesselConnectButton` component.
+2. Through the `useVesselLink` hook.
+
+Here are some example uses of each:
+```jsx
+function App() {
+  return (
+    <VesselConnectButton
+      linkToken={linkToken}
+      onSuccess={(tempToken) => console.log('temp token: ', tempToken)}
+      onClose={() => console.log('closed')}
+      onLoad={() => console.log('loaded')}
+    >
+      Connect your CRM!
+    </VesselConnectButton>
+   )
+}
+```
+```js
+function App() {
+  const { open } = useVesselLink({ 
+    linkToken, 
+    onSuccess: (tempToken) => console.log('temp token: ', tempToken),
+    onClose: () => console.log('closed'),
+    onLoad: () => console.log('loaded')
+  })
+  
+  return <button onClick={() => open()}>Connect your CRM!</button>
+}
+```
+
+A common pattern is to embed each of the Vessel supported integrations directly into your app instead of using the selection modal. This allows for an additional level of white-labeling and will result in the modal opening directly to the given integration's authentication flow. Currently, this is only supported when using the `useVesselLink` hook.
+
+Here's an example of emeding the integrations directly into your application:
+```jsx
+function App() {
+  const { open } = useVesselLink({ 
+    linkToken, 
+    onSuccess: (tempToken) => console.log('temp token: ', tempToken),
+    onClose: () => console.log('closed'),
+    onLoad: () => console.log('loaded')
+  })
+  
+  return (
+    <div>
+      <button onClick={() => open(IntegrationIds.Salesforce)}>Connect Salesforce</button>
+      <button onClick={() => open(IntegrationIds.HubSpot)}>Connect HubSpot</button>
+    </div>
+   )
+}
+```
+~*Note*: `useVesselLink` must only be called once~
```

Comment:
out of curiosity, what happens if they call useVesselLink more than once? e.g. if they're dumb and create a custom button component that calls useVesselLink. If they have n buttons, it would make n calls to useVesselLink

Reply:
then it'll break :'(. Probably a fix somewhere, but don't think it's worth looking into rn